src/applications/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -25,9 +28,13 @@ export class AdditionalInformation extends React.Component {
       <div>
         <strong>Veterans tuition policy:</strong>
         &nbsp;
-        <a href={it.vetWebsiteLink} target="_blank">
+        <button
```

Comment:
I think this one needs to be a link. I tried it out locally, and the URL didn't open until I changed it back to an `<a>` tag in dev tools.

Reply:
Yeah, not sure why I changed this one...